gatsby/content/projects/clients/thunderbird.mdx

```diff
@@ -14,6 +15,7 @@ platforms:
     - Mac
     - Windows
 repo: https://hg.mozilla.org/comm-central
+screenshot: /docs/projects/images/thunderbird.png
 home: https://www.thunderbird.net/
 room: "#maildev:mozilla.org"
```

Comment:
It might be more appropriate to point to our space at `#thunderbird-community:mozilla.org`? Or at the very least at `#thunderbird:mozilla.org`?

Reply:
I think this info is meant for users, so I think either of those makes more sense. 👍

Reply:
I would favour a room to welcome people over a Space, as some Matrix clients don't support them yet. @freaktechnik do you mind updating the room to `#thunderbird:mozilla.org` before I merge this PR :) ?